.github/workflows/combine_scalasteward_prs.yml

```diff
@@ -64,8 +65,12 @@ jobs:
             for pr in ${NEXT_SLICE[@]}
             do
               echo "Bringing in: $pr"
-
               git fetch origin pull/${pr}/head:pr_${pr}_temp
+              git checkout ${NEW_BRANCH}
+              echo "${NEW_BRANCH} is currently at: $(git rev-parse --verify HEAD)"
+              git checkout pr_${pr}_temp
+              echo "pr_${pr}_temp is currently at: $(git rev-parse --verify HEAD)"
+
               git rebase "${NEW_BRANCH}" "pr_${pr}_temp" && EXIT_CODE=$? || EXIT_CODE=$?
```

Comment:
TOL what's this do?

Reply:
Tries to rebase the PR branch onto the new branch being constructed, and then make sure we capture the exit code and continue, even if the rebase fails (necessary because of the `set -euo pipefail` above)